src/testers.jl

```diff
@@ -110,7 +110,8 @@ function test_frule(
         end
         res = frule((NoTangent(), deepcopy(ẋs)...), f, deepcopy(xs)...; deepcopy(fkwargs)...)
         res === nothing && throw(MethodError(frule, typeof((f, xs...))))
-        res isa Tuple || error("The frule should return (y, ∂y), not $res.")
+        msg = "The frule should return (y, ∂y), not $res."
+        @test_msg msg res isa Tuple && length(res) == 2
```

Comment:
Could write this as


```suggestion
        @test_msg msg res isa Tuple{Any,Any}
```

What do you think?